e2e/test/scenarios/dashboard-cards/dashboard-drill.cy.spec.js

```diff
@@ -645,6 +645,41 @@ describe("scenarios > dashboard > dashboard drill", () => {
     });
   });
 
+  it("should keep card's display when doing zoom drill-through from dashboard (metabase#38307)", () => {
+    cy.log("Create a question");
```

Comment:
nit: i think the test is readable without the logs too 👍

Reply:
Good point. Got rid of logs those logs.